api/models.py

```diff
@@ -81,8 +81,12 @@ class DatabaseModel(ModelId):
     def update(self):
         """Method to update model"""
 
+    @classmethod
+    def create_index(cls, collection):
```

Comment:
```suggestion
    def create_indexes(cls, collection):
```
Each model might have several indexes.